tests/conftest.py

```diff
@@ -239,7 +239,14 @@ def run_line(cli_runner, request, patch_tokenstorage):
     for easier debugging.
     """
 
-    def func(line, assert_exit_code=0, stdin=None, matcher=False):
+    def func(
+        line,
+        assert_exit_code=0,
+        stdin=None,
+        match_out=None,
+        match_err=None,
```

Comment:
I have a preference to make it explicit that STDOUT and STDERR are being searched.

```suggestion
        match_stdout=None,
        match_stderr=None,
```